org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/ParallelFileSystemAccess.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 2017, 2020 itemis AG (http://www.itemis.eu) and others.
```

Comment:
@cdietrich @tivervac Kindly confirm the date. The Copyright is missing in the file history. I checked other files that were introduced in the "2.7" release and put the date.  Please suggest if this needs to be changed.

Reply:
should be 2014

Reply:
Updated the year to 2014.